src/index.html

```diff
@@ -5,10 +5,55 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo_Moyo"
+          class="Moyo_img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="index.html" >Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="index.html">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="index.html">Smartphones</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a class="nav__link" href="index.html">Samsung</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="index.html">Smartphones</a>
          </li>
```

Next time pls add blank line between multiple sibling blocks of html